code/modules/modular_computers/file_system/programs/system/file_browser.dm

```diff
@@ -144,12 +150,39 @@
 			return 1
 		var/datum/computer_file/C = F.clone(0)
 		HDD.store_file(C)
+	if(href_list["PRG_encrypt"])
+		. = 1
+		var/obj/item/weapon/computer_hardware/hard_drive/HDD = computer.hard_drive
```

Comment:
Need to check if HDD exists after this, and return if not. See line 135 or 146.

Reply:
Skull's comment is now invalid, check is present